packages/fields/src/actions/duplicate-post.tsx

```diff
@@ -37,8 +37,8 @@ const duplicatePost: Action< BasePost > = {
 		const [ item, setItem ] = useState< BasePost >( {
 			...items[ 0 ],
 			title: sprintf(
-				/* translators: %s: Existing template title */
```

Comment:
Nice catch, thanks! We actually [hide](https://github.com/WordPress/gutenberg/blame/be60bebeff7f6f699fd33d1c35840d9b549cbaa7/packages/editor/src/dataviews/store/private-actions.ts#L137) this action for templates.